### PR TITLE
add fcitx support to chinese layer.

### PR DESCRIPTION
**Pull request by zilongshanren**

### CODE REVIEW

**Comment by StreakyCobra** [comment]
:+1: Thanks for your work :-)

File: layers/chinese/README.org [hunk 1]
```diff
@@ -50,6 +52,19 @@ file. You could call =pyim-dicts-manager= to open up the settings buffer and
...
+  (setq-default dotspacemacs-configuration-layers '((chinese :variables
+                                                             chinese-enable-fcitx t)))
+#+END_SRC
+If you don’t need to type Chinese in minibuffer, you could temporarily disable fcitx in minibuffer
+with the following configuration.
+
+#+BEGIN_SRC emacs-lisp
+  (defun your-layer/post-init-fcitx ()
+    (fcitx-aggressive-setup))
+#+END_SRC
```
Comment (zilongshanren) [inline layers/chinese/README.org]: @TheBB Thanks, I will change it.

File: layers/chinese/packages.el [hunk 2]
```diff
@@ -13,12 +13,11 @@
...
-      find-by-pinyin-dired
-      ace-pinyin
-      pangu-spacing
-      org
-      ))
+      '(
+        find-by-pinyin-dired
+        ace-pinyin
+        pangu-spacing
+        org))
```
Comment (StreakyCobra) [inline layers/chinese/packages.el]: Isn't fcitx package missing here?

Comment (zilongshanren) [inline layers/chinese/packages.el]: @StreakyCobra 
No， i add it only when `chinese-enable-fcitx` is t.

See 

```
(if chinese-enable-fcitx
   (push 'fcitx chinese-packages))
```

**Comment by zilongshanren** [comment]
@TheBB How about this PR now?

**Comment by syl20bnr** [comment]
Thank you :+1: 
Cherry-picked into develop branch, you can safely delete your branch.

**Comment by zilongshanren** [comment]
@syl20bnr  Thanks :+1:

**Comment by colin4124** [comment]
I found it not work in dotspacemacs-editing-style 'hybrid.

Does it have any solutions?

**Pull request closed without merging**